js/src/common/Application.tsx

```diff
@@ -365,9 +365,21 @@ export default class Application {
 
   updateTitle(): void {
     const count = this.titleCount ? `(${this.titleCount}) ` : '';
-    const pageTitleWithSeparator = this.title && m.route.get() !== this.forum.attribute('basePath') + '/' ? this.title + ' - ' : '';
-    const title = this.forum.attribute('title');
-    document.title = count + pageTitleWithSeparator + title;
+    const onHomepage = m.route.get() === this.forum.attribute('basePath') + '/';
+
+    const params = {
+      pageTitle: this.title,
+      forumName: this.forum.attribute('title'),
+      // Until we add page numbers to the frontend, this is constant at 1
+      // so that the page number portion doesn't show up in the URL.
+      pageNumber: 1,
+    };
+
+    const title =
+      onHomepage || !this.title
+        ? app.translator.trans('core.lib.meta_titles.without_page_title', params)
+        : app.translator.trans('core.lib.meta_titles.with_page_title', params);
```

Comment:
These need to use `extractText`

![image](https://user-images.githubusercontent.com/6401250/147487514-044be118-5c07-405c-93b2-4b1afe5f3e9b.png)